app/models/garden.rb

```diff
@@ -1,6 +1,6 @@
 class Garden < ApplicationRecord
   belongs_to :user
-  has_many :plants
+  has_many :plants, dependent: :destroy
```

Comment:
I agree that it should be a dependent destroy, however, I could see users making an argument for wanting to delete a garden and move the plants to a new garden, however I think this is a very unlikely scenario. Good call!